tests/cpu/Config_tests.cpp

```diff
@@ -8676,3 +8676,63 @@ OCIO_ADD_TEST(Config, look_is_noop)
         OCIO_CHECK_ASSERT(proc->isNoOp());
     }
 }
+
+OCIO_ADD_TEST(Config, look_fallback)
+{
+    // Test that the look fallback syntax works for look with missing file.
```

Comment:
Please add a comment to explain what is the meaning of a look fallback `missing_file_look | ` in the unit test. This notation can be used for fallback but it mainly highlights that there are two choices (i.e. `missing_file_look` and `none`) and the precessing takes the first successful.

Reply:
Good point, comment added.